src/main/java/com/razorpay/RazorpayClient.java

```diff
@@ -20,5 +25,14 @@ public RazorpayClient(String key, String secret, Boolean enableLogging) {
     Refunds = new RefundClient(auth);
     Orders = new OrderClient(auth);
     Invoices = new InvoiceClient(auth);
+    Cards = new CardClient(auth);
+    Customers = new CustomerClient(auth);
+    Utility = new Utils(key, secret);
+  }
+
+  public RazorpayClient withHeaders(Map<String, String> headers) {
```

Comment:
`withHeaders` makes sense if you're initializing your client using a builder pattern. This should be `addHeaders`